packages/rum-recorder/src/domain/rrweb-snapshot/snapshot.ts

```diff
@@ -207,7 +185,7 @@ function serializeNode(
         systemId: (n as DocumentType).systemId,
       }
     case n.ELEMENT_NODE:
-      const needBlock = isBlockedElement(n as HTMLElement, blockClass, blockSelector)
+      const needBlock = nodeIsHidden(n)
```

Comment:
Would it make sense to rename all `block` occurrences to `hidden` as well?

Reply:
agreed, I'll add a commit with that rename